src/service/impl/trips/converters.ts

```diff
@@ -68,6 +68,7 @@ function mapToLegacyLeg(leg: Leg_v2): Leg {
       leg.serviceJourney
     ) as ServiceJourney,
     situations: [],
+    bookingArrangements: undefined,
```

Comment:
@rosvik I had to add this because of an issue, do you know if that is OK? Or what that legacy is about.

Reply:
That looks fine! This is to support clients that haven't updated to an app version with OTP2 travel search. It should be removed at some point, but for now, it's just to make sure old clients get somewhat useful data, and don't crash.